### ISSUE TITLE
aggregate chain fail

### ISSUE THREAD

**Issue by stableShip** [underscopeio/sinon-mongoose#8]
use sinon-mongoose to mock aggregate method with chain lookup, show err:
```
var sinon = require('sinon');
require('sinon-as-promised');
require('sinon-mongoose');

   describe("getNewTask", function () {
        var taskQueueMock;
        it("应当返回加密后字符串", function (done) {
            co(function*() {
                try {
                    taskQueueMock = sinon.mock(model.taskQueue);
                    taskQueueMock.expects('aggregate')
                        .chain('lookup')
                        .resolves({
                            "_id" : "57b5fea8bcd0fd2f0df06320"
                            }
                        );
                    var result = yield model.taskQueue.aggregate();
                    console.log(result);
                    done();
                } catch (err) {
                    console.log(err);
                    done(err);
                }
            })
        });
    });
```

error:

```
 TypeError: Attempted to wrap undefined property lookup as function
      at Object.wrapMethod (node_modules/sinon/lib/sinon/util/core.js:106:29)
      at Object.expects (node_modules/sinon/lib/sinon/mock.js:88:27)
      at Object.mock.expects (node_modules/sinon-mongoose/dist/index.js:19:37)
      at Function.chain (node_modules/sinon-mongoose/dist/index.js:12:20)
      at test/taskManager.test.js:54:26
      at next (native)
      at onFulfilled (node_modules/co/index.js:65:19)
      at node_modules/co/index.js:54:5
      at co (node_modules/co/index.js:50:10)
      at Context.<anonymous> (test/taskManager.test.js:50:13)

```

just mock aggregate without chain work

**Comment by gaguirre** [comment]
Hi @stableShip, thanks for reporting the issue.
It's fixed in new release **1.3.0**

**Comment by stableShip** [comment]
:+1:
